Update CS&S email routing (#793)
2i2c-org/team-compass: administration/css.md

- [The CS&S Handbook](https://www.notion.so/CS-S-Handbook-18cd12a6e44c4393857642da6a6b0fdf) is similar to our Team Compass.
- [CS&S Employee Handbook](https://docs.google.com/document/d/1LDN8-iSak391uQC5AzvtzD9dIOmfHg8kihwlvzn8Cy8/edit#heading=h.gjdgxs) is more like an employee policy guide.

```{tip}

> 📧 Please avoid emailing individual CS&S email addresses where possible. This ensures CS&S staff have maximum visibility of incoming email addresses and your query may be resolved more quickly as a result.

Reviewer: ```suggestion
📧 Please avoid emailing individual people within CS&S - instead use group-level e-mails like `fsp@codeforsociety.org`.
This ensures CS&S staff have maximum visibility of incoming email addresses and your query may be resolved more quickly as a result.
```

E-mail address: **`fsp@codeforscience.org`**.

## CS&S HR

For confidential Human Resources needs.

E-mail address: **`hr@codeforscience.org`**.

## CS&S Bills

For sending invoices. This inbox is automated and not monitored.

E-mail address: **`bills@codeforscience.org`**.

## Google Group

We have a dedicated Google Group and e-mail that we use to provide CS&S team members access to our files.

- **`css-admin@2i2c.org`**

This is because they need visibility into many of our operational information for bookkeeping and accountability purposes.
It contains several administrative and organizational leaders of CS&S.
They are all listed as managers, so can add people on their own.

In Google Drive, you may add the `css-admin@2i2c.org` group to the sharing list for a folder or file and they will have access.

## Asana board

CS&S has an Asana board that they occasionally use to track work items.
You can find it at this link:

[CS&S Asana Board](https://app.asana.com/0/1200569652722016/list).

```{note}

> The list above is fairly comprehensive – there is no need to use any other aliases (such as `accouting@codeforscience`, etc.) since they are intended for outgoing use.

Author: ```suggestion
The list above is fairly comprehensive – there is no need to use any other aliases (such as `accounting@codeforscience`, etc.) since they are intended for outgoing use.
```

> The list above is fairly comprehensive – there is no need to use any other aliases (such as `accounting@codeforscience`, etc.) since they are intended for outgoing use.

Reviewer: ```suggestion
The list above is comprehensive – there is no need to use any other aliases (such as `accounting@codeforscience`, etc.) since they are intended for outgoing use.
```
